### PR TITLE
Trigger signing via self-hostet git instance

### PR DESCRIPTION
**Pull request by DaniilKl**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -68,6 +68,13 @@ jobs:
         run: |
           ./meta-dts/scripts/generate-ipxe-menu.sh ${{steps.dts-ver.outputs.DTS_VER}}
           scp -i ~/.ssh/dts-ci-key dts.ipxe builder@10.1.40.2:boot/dts/
+      - name: Trigger signing:
+        shell: bash
+        run: |
+          DTS_VER="${{steps.dts-ver.outputs.DTS_VER}}"
+          git clone https://git.3mdeb.com/3mdeb/dts-release-cicd-pipeline.git
+          git tag `echo $DTS_VER | tr -d "v"`
+          git push origin main --tags
```
Comment (DaniilKl) [inline .github/workflows/ci.yml:77]: Trigger automatic signing via pushing tag with actual version to a repo which is located on self-hosted git instance

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -68,6 +68,14 @@ jobs:
...
           ./meta-dts/scripts/generate-ipxe-menu.sh ${{steps.dts-ver.outputs.DTS_VER}}
           scp -i ~/.ssh/dts-ci-key dts.ipxe builder@10.1.40.2:boot/dts/
+      - name: Trigger signing
+        shell: bash
+        run: |
+          DTS_VER="${{steps.dts-ver.outputs.DTS_VER}}"
+          git clone ssh://git@git.3mdeb.com:2222/3mdeb/dts-release-cicd-pipeline.git
+          cd dts-release-cicd-pipeline
+          git tag `echo $DTS_VER | tr -d "v"`
+          git push origin main --tags
```
Comment (DaniilKl) [inline .github/workflows/ci.yml:78]: Here operations on private repo is performed. I assume that the server where the runner runs this pipeline has permissions to manipulate on our private repos.

**Comment by DaniilKl** [comment]
This PR is ready to review. 

@TomaszAIR be aware that **I have not tested those changes** cos I do not know if it is a good idea to trigger this pipeline.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (TomaszAIR) [reply .github/workflows/ci.yml:78]: @DaniilKl so we need an additional SSH key in this pipeline which will allow this cloning. Did you test this? While testing we do not need pushing any tags, just cloning will be enough.

Comment (DaniilKl) [reply .github/workflows/ci.yml:78]: @TomaszAIR I did not tested it ([comment](https://github.com/Dasharo/meta-dts/pull/41#issuecomment-1736998331)). 

Yep, we need some key to get access to our repos.

Comment (TomaszAIR) [reply .github/workflows/ci.yml:78]: @DaniilKl you should have access to both (gitea and builder), please add SSH key via secret, test it and I will later replace that key with a valid one

Comment (TomaszAIR) [reply .github/workflows/ci.yml:78]: @DaniilKl jop runs on any tag pushed so you could e.g. bump version to 1.2.10-rc1, then create such tag and push it. Or, even better, just for tests comment out any commands that build/clone artifacts to the FTP server.

**Pull request merged**